api/README.md

```diff
@@ -1,5 +1,33 @@
 # RealWorld API Spec
 
+## How to use the supplied Postman configuration
+
+First ensure you have `newman` installed, by running `newman -v`. If
+that fails, ensure you have a recent NodeJS installed, then:
+
+```
+sudo npm install -g newman
```

Comment:
Typically I suggest not using sudo when installing node modules, it can lead to downstream permissions issues. I would suggest removing the sudo from here.